pgjdbc/src/test/java/org/postgresql/test/jdbc2/DriverTest.java

```diff
@@ -62,7 +62,7 @@ public void testAcceptsURL() throws Exception {
     verifyUrl(drv, "jdbc:postgresql://127.0.0.1/anydbname", "127.0.0.1", "5432", "anydbname");
     verifyUrl(drv, "jdbc:postgresql://127.0.0.1:5433/hidden", "127.0.0.1", "5433", "hidden");
     verifyUrl(drv, "jdbc:postgresql://[::1]:5740/db", "[::1]", "5740", "db");
-    verifyUrl(drv, "jdbc:postgresql:?service=testsrv", "srvhost", "5434", "srvdb");
+    verifyUrl(drv, "jdbc:postgresql://?service=testsrv", "srvhost", "5434", "srvdb");
```

Comment:
Cheers! I don't want to get too picky here, but can you add some tests as well for (a) the three-slash case, and (b) failure?

Reply:
You're right.

Actually. `jdbc:postgresql://?service=testsrv` triggers an error. `///` is fine. I don't want to rewrite all `parseURL` for `//`.